.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Parsers/SpanishTimeParserConfiguration.cs

```diff
@@ -42,16 +56,19 @@ public void AdjustByPrefix(string prefix, ref int hour, ref int min, ref bool ha
             var deltaMin = 0;
             var trimmedPrefix = prefix.Trim();
 
-            // @TODO move hardcoded values to resources file
-            if (trimmedPrefix.StartsWith("cuarto", StringComparison.Ordinal) || trimmedPrefix.StartsWith("y cuarto", StringComparison.Ordinal))
+            if (QuarterTokenRegex.IsMatch(trimmedPrefix))
             {
-                deltaMin = 15;
-            }
-            else if (trimmedPrefix.StartsWith("menos cuarto", StringComparison.Ordinal))
-            {
-                deltaMin = -15;
+                var match = QuarterTokenRegex.Match(trimmedPrefix);
+                if (match.Groups["neg"].Success)

```

Comment:
The hardcoded "neg" should be a constant along with the other named group identifiers. Especially if it's a named group used for the same purpose in other languages.

Reply:
ok, added to Constants